tools/wpt/browser.py

```diff
@@ -656,8 +656,43 @@ def find_binary(self, venv_path=None, channel=None):
         self.logger.warning("Unable to find the browser binary.")
         return None
 
-    def find_webdriver(self, channel=None):
-        return find_executable("chromedriver")
+    def find_webdriver(self, channel=None, browser_binary=None):
+        chromedriver_binary = find_executable("chromedriver")
+        if not chromedriver_binary:
+            return chromedriver_binary
+
+        # Check that the chromedriver matches the installed Chrome. We accept
+        # up to one version behind, to allow for using a tip-of-tree Chromium
+        # build right after branch point.
```

Comment:
Nit: maybe move this to before `chromedriver_major`.

In fact, I wonder if we need this at all. The version selection logic will eventually fall back to using the ToT **Chromium** build of chromedriver, which should have the matching major version.

Reply:
Ah neat, I hadn't considered that. Updated to be a direct version match.